run.ts

```diff
@@ -56,6 +59,16 @@ async function main() {
     }
 }
 
+async function lastFiveCommands() {
```

Comment:
Add a comment to explain what this function does:

```suggestion
// Retrieve the last 5 run commands from `~/.zsh_history` (if it exists).
async function lastFiveCommands() {
```

In writing the comment ,you'll find it's a bit tricky to explain what it's doing :) Which _can_ be a good sign it should be split into two functions. In this case, it's probably fine as one. Once it becomes separate discover+parse logic you may want to refactor it.

Reply:
Done :)